src/content/basics/automatic-termination-of-bad-nodes/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Automatic termination of unhealthy nodes
-description: A general description of a feature that can detect and terminate unhealthy nodes in the tenant cluster.
-date: 2020-11-05
+description: When worker nodes get into a state where they do not respond to health checks. Here we explain how you can activate automatic termination of such nodes.
```

Comment:
this description is working for now but in the future we want to expand the portfolio, and saying that `a state where they do not respond to health checks.` will be false as we also want to terminate nodes that have filled docker or kubelet disk but they can still respond to health checks

Reply:
Or some other cases like  corrupted networks

Reply:
so definition `unhealthy nodes` feels more accurate description

Reply:
Done